report/28_Process-Logging.md

```diff
@@ -1 +1,38 @@
-What do you log in your systems and how do you aggregate logs?
+### Logging
+
+Logging output from processes is important in order to know what is going on in the system.
+If a program in no way outputs any information, then it is hard to diagnose faults or detect attacks, which can make fixing those things harder.
+Since the entire website is written in .NET, it makes sense to rely on already existing logging modules to assemble and aggregate system information.
+One such module, called [Serilog](https://github.com/serilog/serilog), is what this project uses for this cause. 
+
+Serilog is a powerful low-overhead diagnostic logging library, capable of logging information in familiar log levels to many different log formats and log aggregators.
+Using it is simple, as setup simply involves initializing the logger singleton with write locations (such as console, file, or Elasticsearch), after which the project, through dependency injection, is capable of referring to the same logger when needed.
+Logging in code becomes as easy as calling a method through the injected interface. 
+
+We log all API call events, including but not limited to:
+
+- `/latest`.
+- Getting messages by all/specific user(s), along with a count of returned messages.
+- Posting twits, both for the successful and failure paths.
+- Registering a user, including cases with mail collisions and the like.
+- Getting messages for follows, including count (again).
+- Any of the calls made when unauthorized.
+
+This logging is done all with the information level, and it makes sense: 
```

Comment:
Does it make sense? Also Information level needs some context.

Reply:
Yeah, der er nok noget der skal omskrives her, det der kolon er også lidt weird.